lib/membrane/rtp/payload_type_resolver.ex

```diff
@@ -31,13 +31,15 @@ defmodule Membrane.RTP.PayloadFormatResolver do
     end
   end
 
+  @spec keyframe_detector(atom()) :: {:ok, function()} | :error
   def keyframe_detector(encoding) do
     case PayloadFormat.get(encoding).keyframe_detector do
       nil -> :error
       keyframe_detector -> {:ok, keyframe_detector}
     end
   end
```

Comment:
I guess this is unused and to be refactored, so I'd remove it

Reply:
It is used in RTC Engine